static/app/components/events/eventReplay/replayPreview.tsx

```diff
@@ -45,9 +49,29 @@ function ReplayPreview({orgSlug, replaySlug, event}: Props) {
   }, [eventTimestamp, startTimestampMs]);
 
   if (fetchError) {
+    const reasons = [
+      t('The replay is still processing'),
+      t('The replay has been deleted by a member in your organization'),
+      t('There is an internal systems error'),
+    ];
+
     return (
       <Alert type="info" showIcon data-test-id="replay-error">
-        {t('The replay associated with this event could not be found.')}
+        <p>
+          {tct(
+            'The replay for this event cannot be found. [link:Read the docs to understand why]. This could be due to a few of reasons:',
```

Comment:
```suggestion
            'The replay for this event cannot be found. [link:Read the docs to understand why]. This could be due to a few reasons:',
```